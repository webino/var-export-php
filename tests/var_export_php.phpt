<?php
/**
 * Webino™ (http://webino.sk)
 *
 * @noinspection PhpUnhandledExceptionInspection
 *
 * @link        https://github.com/webino/var-export-php
 * @copyright   Copyright (c) 2019 Webino, s.r.o. (http://webino.sk)
 * @author      Peter Bačinský <peter@bacinsky.sk>
 * @license     BSD-3-Clause
 */

namespace Webino;

use Tester\Assert;
use Tester\Environment;

Environment::setup();

$var = ['foo' => ['bar' => 'baz']];

$code = var_export_php($var);

Assert::matchFile('var_export_php.expected', $code);
