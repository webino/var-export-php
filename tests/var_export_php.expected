<?php return array (
  'foo' =>
  array (
    'bar' => 'baz',
  ),
);
